readthedocs/projects/forms.py

```diff
@@ -43,11 +43,11 @@ def save(self, commit=True):
 
 class ProjectTriggerBuildMixin(object):
 
-    """Mixin to trigger build on form save
+    '''Mixin to trigger build on form save
 
     This should be replaced with signals instead of calling trigger_build
     explicitly.
-    """
+    '''
```

Comment:
This reverts some previous clean up to follow pep257. When we turn on pep257 with the next round of linting, triple single quotes will be invalid. Same goes for the rest of the single quote changes below.